### PR TITLE
crypto/bio: drop float formating for UEFI

### PR DESCRIPTION
**Pull request by kraxel**

### CODE REVIEW

File: crypto/bio/bio_print.c [hunk 1]
```diff
@@ -293,6 +298,16 @@ _dopr(char **sbuffer,
...
                     return 0;
                 break;
+#else
+            case 'f':
+            case 'E':
+            case 'e':
+            case 'G':
+            case 'g':
+                /* not implemented for UEFI */
+                ASSERT (FALSE);
```
[the remark this answers is not in the document]
Comment (paulidale) [reply crypto/bio/bio_print.c:308]: We don't have asserts in the library.  An error return here is appropriate.

**Comment by kraxel** [comment]
The test failure looks like some temporary network issue, is there some way to re-run that test?

**Review by t8m — APPROVED** [review]

File: crypto/bio/bio_print.c [hunk 2]
```diff
@@ -293,6 +298,17 @@ _dopr(char **sbuffer,
...
+#else
+            case 'f':
+            case 'E':
+            case 'e':
+            case 'G':
+            case 'g':
+                /* not implemented for UEFI */
+                ERR_raise(ERR_LIB_BIO, ERR_R_UNSUPPORTED);
+                return 0;
+                break;
```
Comment (paulidale) [inline crypto/bio/bio_print.c:310]: Remove this line please.  It's unreachable and compilers/static analysers will complain.

**Review by paulidale — APPROVED** [review]
Approved with the line deleted.

**Review by paulidale — APPROVED** [review]

**Comment by t8m** [comment]
Merged to master and 3.0 branches. Thank you for your contribution.

**Pull request closed without merging**